build_image/docker/agent/ansible/Dockerfile.in

```diff
@@ -10,8 +10,9 @@ ARG uid=1000
 ARG gid=1000
 
 RUN apt-get update                                          && \
+    apt-get upgrade     &&\
     apt-get install -y bash curl python-dev sudo               \
-    python-pip build-essential openssh-client libffi-dev       \
+    python3-pip build-essential openssh-client libffi-dev       \
     libssl-dev                                              && \
     pip install --upgrade pip ansible pyyaml                && \
```

Comment:
Should we use pip3 here?

Reply:
If you don't use it, it will prompt that python-pip is not found.